pallets/gear-program/src/lib.rs

```diff
@@ -167,7 +167,7 @@ pub mod pallet {
     use sp_runtime::DispatchError;
 
     /// The current storage version.
-    pub(crate) const PROGRAM_STORAGE_VERSION: StorageVersion = StorageVersion::new(10);
+    pub(crate) const PROGRAM_STORAGE_VERSION: StorageVersion = StorageVersion::new(13);
```

Comment:
why 13, not 11?

Reply:
Partly because CodeMetadata has been replaced by a completely different structure, partly because of simplicity